doc/whats_new.rst

```diff
@@ -157,6 +157,8 @@ Bug
 API
 ~~~
 
+- Deprecation of ``annot`` and ``annotmap`` parameters in :meth:`mne.io.edf.RawEDF` by `Joan Massich`_ (`#5580 <https://github.com/mne-tools/mne-python/pull/5580>`_)
```

Comment:
Sorry I should have been clearer, I don't think we should link to GitHub issues here for 0.17 for consistency but instead maybe decide to start doing it in 0.18